.github/workflows/stale.yml

```
@@ -0,0 +1,19 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'Stale issue message'
```

Shouldn't this (and the line below) be the actual message that will be posted to stale issues/prs?

---

TBH, I just added the default action template, so yes you're right

```suggestion
        stale-issue-message: 'Potential stale issue'
```

Or something like that?